### DISCUSSION TITLE
How can we stop the camera after read the QR code?

### DISCUSSION THREAD

**Discussion by satyapratap-dev** [cozmo/jsQR#199] [Q&A]
Hi Team,
Can we stop the camera/video once QR code scans completed? Let's say I am trying to capturing specific type format QR codes(like emails/mobile number) only. so, if any one of the QR format matches then it should quit/close the QR code scanning(close the camera). Please advise.

**Comment by ericgarzagzz** [comment]
You can use `video.srcObject.stop();` using the example of [https://cozmo.github.io/jsQR/](url). You're going to need to append to `stream` this object.

`
stream.stop = function () {
                this.getAudioTracks().forEach(function (track) {
                    track.stop();
                });
            };
`

I also recommend you to clean up video references and detach them from the DOM when you are no longer using them (for the sake of ram usage).
